src/app/Components/email-preview/email-preview.component.ts

```diff
@@ -55,4 +57,10 @@ export class EmailPreviewComponent implements OnInit {
       });
     }
   }
+  openEmailDetail(data: any) {
+    this.active = data._id;
```

Comment:
in the real app, the argument the event sends could be `undefined` so make an exception for it at the beginning:
`if (!dataId) {
console.log('error_message');
return;
}`